apps/dash-salesforce-crm/README.md

```diff
@@ -0,0 +1,64 @@
+# Dash Salesforce CRM
+
+This is a demo of the Dash interactive Python framework developed by [Plotly](https://plot.ly/).
+
+Dash abstracts away all of the technologies and protocols required to build an interactive web-based application and is a simple and effective way to bind a user interface around your Python code.
+
+To learn more check out our [documentation](https://plot.ly/dash).
+
+## Getting Started
+
+### Running the app locally
+
+First create a virtual environment with conda or venv inside a temp folder, then activate it.
+
+```
+virtualenv venv
+
+# Windows
+venv\Scripts\activate
+# Or Linux
+source venv/bin/activate
+
+```
+
+Clone the git repo, then install the requirements with pip
+
+```
+
+git clone https://github.com/plotly/dash-sample-apps
+cd dash-sample-apps/apps/dash-salesforce-crm
+pip install -r requirements.txt
+
+```
+
+To run the app, please create a SalesForce developer account (link is in the `About the App` section)and put the credentials in the `secrets.example.sh` file.
```

Comment:
why would they have to rename the file to `secrets.sh`? The command is `source secrets.example.sh` to provide the login info right?

Reply:
`secrets.example.sh` is meant to be like a template file. If they modify that file, they might push the credentials to github. However, if they modify the `secrets.sh` file, it won't matter since it's `.gitignore`.

Reply:
Makes sense.